scrapers/JavLibrary_python.py

```diff
@@ -396,7 +396,7 @@ def jav_search_by_name(html, xpath):
             "title": jav_title[count],
             "url":
             f"https://www.javlibrary.com/en/{jav_url[count].replace('./', '')}",
-            "image": f"https:{jav_image[count]}"
+            "image": jav_image[count]"
```

Comment:
Was it always like that or did they change something in the layout?
The comment in line 391 shows a sample image url without the schema (https:).
Would it be safer to use something like 
```python
            "image": re.sub("^//","https://",jav_image[count])
```
or is it not necessary ?

Reply:
Yea would be better, i think the layout has changed or when I made this feature I was blind.